storage-api/src/test/org/apache/hive/common/util/TestTxnIdUtils.java

```diff
@@ -190,5 +191,15 @@ public void testCompareWriteIds() throws Exception {
         new ValidReaderWriteIdList("default.table2", new long[] {8,10,11}, new BitSet(), 11)),
     -1);
 
+    ValidWriteIdList a =
```

Comment:
putting a lot of cases into the same method is not really development friendly - next time please put new testcases into new test methods

Reply:
Sure @kgyrtkirk . I will keep your suggestion in mind for future commits. Thanks for the review and approval.